### PR TITLE
Enhance documentation

### PR DESCRIPTION
**Pull request by samuelspagl**
## 📚 Enhance Documentation

**Closes** #18 

This PR aims to clarify and enhance the current documentation. Sites were added and removed and grouped differently.

Thanks a lot to @tjorim for reviewing the changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,19 @@
-# HomeAssistant: Samsung Soundbar Integration
-
-> Yet another Samsung Soundbar Integration (YASSI)
+# YASSI: Yet Another Samsung Soundbar Integration (for HomeAssistant)
```
Comment (tjorim) [inline README.md:1]:
```suggestion
# YASSI: Yet Another Samsung Soundbar Integration (for Home Assistant)
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
+
+Welcome to YASSI, the HomeAssistant integration designed to bring comprehensive control over your Samsung Soundbar into your smart home ecosystem.
```
Comment (tjorim) [inline README.md:3]:
```suggestion
Welcome to YASSI, the Home Assistant integration designed to bring comprehensive control over your Samsung Soundbar into your smart home ecosystem.
```

File: docs/content/1.getting-started/0.index.md [hunk 3]
```diff
@@ -1,30 +1,42 @@
...
 
+#### 🌟 Official Repository (Coming Soon)
+The Yassi integration will be available through the official HACS repository shortly. Stay tuned for updates.
 
-### HACS (custom repository)
+#### ➕ Custom Repository
+In the meantime, you can manually add this repository to HACS:
+1. Navigate to HACS in your HomeAssistant.
+2. Go to 'Settings' and add this repository URL as a custom repository.
+3. Search for 'Yassi' under integrations and proceed with the installation.
```
Comment (tjorim) [inline docs/content/1.getting-started/0.index.md:16]:
```suggestion
1. Click the following button and 'open link':
[![Open your Home Assistant instance and open a repository inside the Home Assistant Community Store.](https://my.home-assistant.io/badges/hacs_repository.svg)](https://my.home-assistant.io/redirect/hacs_repository/?owner=samuelspagl&repository=ha_samsung_soundbar&category=integration)
2. Click 'add' to add the custom repository.
3. Download 'Yassi' and reboot Home Assistant.
```

**Review by tjorim — COMMENTED** [review]
Are you planning to rename this integration to YASSI? Otherwise users will need to search for the 'samsung soundbar' integration when setting it up in Home Assistant.

**Comment by samuelspagl** [comment]
> Are you planning to rename this integration to YASSI? Otherwise users will need to search for the 'samsung soundbar' integration when setting it up in Home Assistant.

I would probably rename it to "YASSI: Yet Another Samsung Soundbar Integration" which will result that you find the integration by either typing "Yassi" or "Samsung Soundbar".

Or is there something against this approach?

File: docs/content/1.getting-started/0.index.md (hunk 3 above)
Comment (samuelspagl) [reply docs/content/1.getting-started/0.index.md:16]: Didn't know that a redirect like that is possible, cool. Thanks!

**Comment by samuelspagl** [comment]
@tjorim I updated it again, mind taking a look? :) (Do you have access to the preview site?)

File: docs/content/1.first-things-first/2.issues-and-other-things.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# ‼️ Issues and other things
+
+As the creator of this personal and fun project, I am thrilled to see people using it. While I won’t always have immediate availability to address every request, I’ll do my best to fix issues and implement features. Thanks a lot in advance! 🙌
+
+Here are some best practices to help me help you:
+
+1. 🐞 GitHub Issues: For any issues or bugs, please submit them via GitHub Issues. ([🔗 click here](https://github.com/samuelspagl/ha_samsung_soundbar/issues/new))
+2. 📋 Provide Details: Include essential information:
+    - HomeAssistant OS Version
```
Comment (tjorim) [inline docs/content/1.first-things-first/2.issues-and-other-things.md:9]:
```suggestion
    - Home Assistant OS Version
```
Do you really want to know the OS version, not the core version?

File: docs/content/1.first-things-first/2.issues-and-other-things.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    - Samsung Soundbar Model
+    - Other Relevant Details
```
Comment (tjorim) [inline docs/content/1.first-things-first/2.issues-and-other-things.md:11]: Maybe mention debug logs as well.

File: docs/content/2.features/0.index.md [hunk 6]
```diff
@@ -1,35 +1,44 @@
...
-- `button`
-  - `night_mode` [*read, write*]
-  - `voice_amplifier` [*read, write*]
-  - `bass_mode` [*read, write*]
-
-- `image`
-  - `media_coverart` [*read*]
+# ✨ Features Overview
+
+Yassi allows you to retrieve and set the status of various features on your Samsung Soundbar. Below is a breakdown of capabilities organized by entity type.
```
Comment (tjorim) [inline docs/content/2.features/0.index.md:3]: I think we're missing the Volume level `sensor`.

**Review by tjorim — COMMENTED** [review]
Left some comments, only not sure about the YASSI vs Yassi inconsistency.

**Review by tjorim — APPROVED** [review]

**Pull request merged**